AndroidTestAutomationStarter/kotlinapi/src/main/java/com/novoda/kotlinapi/database/InMemoryDatabase.kt

```diff
@@ -0,0 +1,76 @@
+package com.novoda.kotlinapi.database
+
+import com.novoda.kotlinapi.model.Article
+import com.novoda.kotlinapi.model.Category
+
+class InMemoryDatabase {
+    private val descriptionOne = "Bacon ipsum dolor amet in meatball shank prosciutto, anim pork loin nisi bresaola. Buffalo beef salami shank pancetta tail sunt prosciutto, excepteur drumstick ground round anim nulla sint jerky."
+    private val descriptionTwo = "Velit cupidatat deserunt pork chop cillum, sirloin elit chicken brisket officia excepteur chuck."
+    private val descriptionThree = "Jerky laboris veniam, laborum pancetta labore sint buffalo dolor quis strip steak sirloin ground round."
+    private val descriptionFour = "Chuck consectetur chicken drumstick short ribs in tenderloin frankfurter eiusmod lorem. Dolore magna excepteur venison. Tempor enim rump ut fugiat minim esse."
+    private val articles = hashMapOf(
+            0 to arrayListOf(
+                    Article(id = 4, title = "Hat Green", price = "$18", image = "shirt1", productDescription = descriptionOne),
```

Comment:
Maybe give names that correspond to the products? If we don't have images now we can leave that for later and only do the text